src/mixins/document-mixin.ts

```diff
@@ -38,4 +49,24 @@ export default class DocumentMixin extends Vue {
         return error.response
       })
   }
+
+  /**
+   * Checks whether pdf file is using specified page size by checking width and height of 1st page of pdf file
+   * @param file pdf file to be checked
```

Comment:
Is there a type that can be used?

Reply:
updated to include `File` type.  i.e. `async isPageSize (file: File, pageSize: PdfPageSize): Promise<boolean> {`